main.yml

```diff
@@ -228,6 +228,21 @@ rhel:
       - id: remediations
   top_level: true
 
+ruledev:
+  title: Rule Development
+  channel: '#flip-mode-squad'
+  deployment_repo: https://github.com/RedHatInsights/ruledev-frontend-build
+  frontend:
+    title: Rule Development
+    paths:
+      - /ruledev
+    sub_apps:
```

Comment:
Is this an app group? This field should only exist if you want a sub app that exists under the `ruledev` app.